indico/modules/events/contributions/controllers.py

```diff
@@ -16,10 +16,31 @@
 
 from __future__ import unicode_literals
 
+from flask import flash, request
+
+from indico.modules.events.contributions.models.contributions import Contribution
+from indico.modules.events.contributions.operations import create_contribution, update_contribution, delete_contribution
+from indico.modules.events.contributions.forms import ContributionForm
 from indico.modules.events.contributions.views import WPManageContributions
+from indico.web.flask.templating import get_template_module
+from indico.web.forms.base import FormDefaults
+from indico.web.util import jsonify_data, jsonify_template
+from indico.util.i18n import _
 from MaKaC.webinterface.rh.conferenceModif import RHConferenceModifBase
 
 
+def _get_contribution_list_args(event):
+    contribs = event.contributions.filter_by(is_deleted=False).all()
```

Comment:
`.order_by(db.func.lower(Contribution.title))` so we have some default order? Could also be by ID. dunno what's better